specification/storage/resource-manager/Microsoft.Storage/stable/2019-04-01/file.json

```diff
@@ -63,7 +63,7 @@
         "tags": [
           "FileService"
         ],
-        "operationId": "FileServices_PutServiceProperties",
+        "operationId": "FileServices_SetServiceProperties",
```

Comment:
I suppose this should be considered as a swagger breaking change, which should not happen in a stable version of a swagger

Reply:
This API is added several days ago with PR https://github.com/Azure/azure-rest-api-specs/pull/6892
No SRP SDK released with it, so change the operationId now won't take SDK breaking change.

Besides that, since the name "FileServices_PutServiceProperties" is not aligned with Blob service API name, we should make it aligned. If we change it now, not breaking on SDK; if we change it later after SDK release with the API, it will take breaking to SDK later.

Reply:
I understand. But according to the [documentation](https://github.com/Azure/adx-documentation-pr/blob/master/swagger/swagger-workflow-for-reviewers.md) of swagger reviewing, this kind of changes should not be allowed. Let us see how ARM team will respond to this.